VultisigApp/VultisigApp/Views/Send/SendCryptoView.swift

```diff
@@ -54,8 +56,12 @@ struct SendCryptoView: View {
                     backButton
                 }
             }
-            
-            if sendCryptoViewModel.currentIndex==3 {
+            if showFeeSettings {
```

Comment:
Is there a reason why we don't add this to MacOS?

Reply:
just want to make it in separate PR